kuma/javascript/src/.eslintrc.js

```diff
@@ -0,0 +1,71 @@
+module.exports = {
```

Comment:
I created a new eslint config for this directory because this is mainly where we are using prettier and react. The eslint config above has indentation rules that are not always compatible with what prettier does, but I don't want to remove them since we still have JS code that does not use prettier. So instead, I'm proposing that we have a different eslint config for the new code we're writing here.